text/0000-extern-types.md

```diff
@@ -59,7 +59,7 @@ extern {
 }
 ```
 
-These types are FFI-safe. They are also DSTs, meaning that they do not implement `Sized`. Being DSTs, they cannot be kept on the stack and can only be accessed through pointers.
+These types are FFI-safe. They are also DSTs, meaning that they do not implement `Sized`. Being DSTs, they cannot be kept on the stack, can only be accessed through pointers and references and cannot be moved from.
```

Comment:
I'd like to note that references are also pointers - you can say "can only be accessed through pointers", or you can say "raw pointers and references".